documentation/documentation/admin/installing-plv8-windows.md

```diff
@@ -26,10 +26,14 @@ You can download the appropriate binaries for PostgreSQL 9.5/PostgreSQL 9.6 from
 
 [PLV8-binaries-for-PostgreSQL-10-windows-both-32-bit-and-64-bit](http://www.postgresonline.com/journal/archives/379-PLV8-binaries-for-PostgreSQL-10-windows-both-32-bit-and-64-bit.htmll)
 
+[PLV8-binaries-for-PostgreSQL-9.4-to-12-windows-64-bit](http://updates.xtuple.com/updates/plv8/win/xtuple_plv8.zip)
```

Comment:
Rename this as `xTuple-PLV8-binaries-for-PostgreSQL-9.4-to-12-windows-64-bit`. Also change the name from `Xtuple` to `xTuple` all places where it is mentioned.